spec/models/bet_option_spec.rb

```diff
@@ -48,9 +53,14 @@
     end
 
     context 'New England wins' do
-      let(:option1) { BetOption.create(option_text: 'Atlanta wins') }
+      let(:option1) do
+        BetOption.create(option_text: 'Atlanta wins',
+                         bet: bet1)
```

Comment:
In your bet_spec.rb file you are one lining a bet option with two parameters. Pick one syntax style and roll with it.